spec/ios_lint_localizations_spec.rb

```diff
@@ -54,6 +54,34 @@
   end
 
   context 'Linter' do
+    def run_l10n_linter_test(data_file)
+      # Arrange: Prepare test files
+      test_file = File.join(File.dirname(__FILE__), 'test-data', 'translations', "test-lint-ios-#{data_file}.yaml")
+      yml = YAML.load_file(test_file)
+
+      files = yml['test_data']
+      FileUtils.mkdir_p(@test_data_dir)
+      files.each do |lang, content|
+        lproj = File.join(@test_data_dir, "#{lang}.lproj")
+        FileUtils.mkdir_p(lproj)
+        File.write(File.join(lproj, 'Localizable.strings'), content) unless content.nil?
+      end
+
+      # Act
+      # Note: We will install SwiftGen in vendor/swiftgen if it's not already installed yet, and intentionally won't
+      #       remove this after the test ends, so that further executions of the test run faster.
+      #       Only the first execution of the tests might take longer if it needs to install SwiftGen first to be able to run the tests.
+      install_dir = "vendor/swiftgen/#{Fastlane::Helper::Ios::L10nLinterHelper::SWIFTGEN_VERSION}"
+      result = run_described_fastlane_action(
+        install_path: install_dir,
+        input_dir: @test_data_dir,
+        base_lang: 'en'
+      )
+
+      # Assert
+      expect(result).to eq(yml['result'])
+    end
+
```

Comment:
Nitpick: I liked this at the bottom of the file so that a reader would scroll to the tests first, and only dive into this implementation detail if they needed to. Would it make sense to move it at the end of the `context` block?

_Fine by me to disregards, in particular since there's no other comment from me at this time._

Reply:
Fair point, but given that all other specs which declare helper `def` functions in a similar way already also have them at the top (e.g. [here](https://github.com/wordpress-mobile/release-toolkit/blob/4fc2e0c52f730268fb81dd01c39a9029b303a947/spec/ios_extract_keys_from_strings_files_spec.rb#L8-L13) or [here](https://github.com/wordpress-mobile/release-toolkit/blob/08602aa689138061d278bbc9f3658c7332054f3c/spec/ios_download_strings_files_from_glotpress_spec.rb#L14-L43))… I figure it would make sense to keep consistency here 🤷

Reply:
> Fair point, but given that all other specs which declare helper `def` functions in a similar way already also have them at the top (e.g. [here](https://github.com/wordpress-mobile/release-toolkit/blob/4fc2e0c52f730268fb81dd01c39a9029b303a947/spec/ios_extract_keys_from_strings_files_spec.rb#L8-L13) or [here](https://github.com/wordpress-mobile/release-toolkit/blob/08602aa689138061d278bbc9f3658c7332054f3c/spec/ios_download_strings_files_from_glotpress_spec.rb#L14-L43))… I figure it would make sense to keep consistency here 🤷

Good point. I wasn't aware of that 👍